test/__snapshots__/index.test.js.snap

```diff
@@ -20,16 +20,16 @@ alias
 Build \\"aliasMapping\\" to dist:
 62 B: alias-mapping.js.gz
 46 B: alias-mapping.js.br
-62 B: alias-mapping.esm.js.gz
-46 B: alias-mapping.esm.js.br
+61 B: alias-mapping.esm.js.gz
```

Comment:
as per side-discussion, this size decrease is due to custom property name mappings being ignored.